config.py

```diff
@@ -35,6 +35,7 @@
 SRV_NAME = config("SRV_NAME", default="osu!Akatsuki")
 
 DISCORD_ADMIN_HOOK = config("DISCORD_ADMIN_HOOK")
+BEATMAP_UPDATE_HOOK = config("BEATMAP_UPDATE_HOOK")
```

Comment:
New env var needed in score-service-k8s

Reply:
Added to both production & production-k8s linking to: https://discord.com/channels/365406575893938177/1242905789035708517